doc/proposals/tls-utilities.md

```diff
@@ -0,0 +1,159 @@
+# TLS Utilities proposal
+
+## Background:
+
+A typical workflow of an operator that is written using the Operator-SDK involves watching a  Custom Resource Definition (CRD), which specifies the desired state of the cluster, and making changes to the current state of the cluster to achieve the desired state.
+
+Each user specified CRD represents a self contained system. Based on this, we can assume that any TLS assets generated are specific to each system which then is also specific to the corresponding CRD.
```

Comment:
Do you mean CR instead of CRD? Since there can be only one crd but many CRs. And each TLS asset is associated with on CR.

Reply:
I went ahead with the assumption that the SDK currently supports only one CR hence one CRD. I could add a line clarifying what a CR and CRD are and make the difference clear